src/images/badges/built-for-developers.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is not rendering the typography correctly for `Built For`. Make sure to outline your text before creating and submitting the SVG as embedded type doesn't work.